Docker don't run

Hello @lafin !
Firstly, i filled credentials in docker file.
I use your setup manual and docker run has failed with: 
```javascript
module.js:340
    throw err;
          ^
Error: Cannot find module '/var/apps/yts/app.js'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:902:3
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,11 +1,6 @@
 FROM ubuntu:14.04
 MAINTAINER lafin <kuvakin@gmail.com>
 
-RUN echo \
-"deb http://mirror.yandex.ru/ubuntu/ trusty main restricted universe multiverse\n\
-deb http://mirror.yandex.ru/ubuntu/ trusty-security main restricted universe multiverse\n\
-deb http://mirror.yandex.ru/ubuntu/ trusty-updates main restricted universe multiverse\n" > /etc/apt/sources.list
-
 RUN apt-get update -y
 RUN apt-get install -y git nodejs npm vim
 RUN useradd -MUs /bin/bash node
@@ -31,4 +26,4 @@ EXPOSE 3000
 
 RUN chown node:node /var/apps/yts/ -R
 USER node
-CMD nodejs /var/apps/yts/app.js
+CMD nodejs /var/apps/yts/server/app.js

